### ISSUE TITLE
Staging and unstaging hunks in `project_diff_preview` mode doesn't work

### ISSUE THREAD

**Issue by Ternvein** [tanvirtin/vgit.nvim#346]
### Description

In `project_diff_preview` mode using default `gs` and `gu` keymaps leads to staging and unstaging the whole file (that is, keymap degrades to just `s` and `u`). A quick glance into the code led me to think that `gs` and `gu` keymaps registered for diff view, but in `project_diff_preview` mode default selected window is list view. I couldn't quickly find any way to change the active window, standard `C-w <some>` doesn't help (in fact, it bugs the VGit popup even more - it becomes unresponsive except for `Esc` to quit).

- nvim --version output:
NVIM v0.9.1
Build type: Release
LuaJIT 2.1.1692616192
- Operating system: Archlinux

**Comment by tanvirtin** [comment]
Yeah, unfortunately, this is a user interface constraint and without knowing what hunk the user is focused it is difficult to stage/unstage the part. 

You can either cycle through the windows or utilize the mouse.

![Image](https://github.com/user-attachments/assets/cea6763e-1451-47d4-9a3a-1e1ad0f74fbc)

Curious to know about the screen unresponsiveness. Please try the latest major `v1.0.0` and feel free to open this up.
